source/_integrations/balboa.markdown

```diff
@@ -0,0 +1,53 @@
+---
+title: "Balboa Spa"
+description: "Instructions on how to integrate Balboa Spa WiFi within Home Assistant."
+ha_category:
+  - Climate
+ha_release: 2021.12
+ha_iot_class: Local Push
+ha_domain: balboa
+ha_platforms:
+  - climate
+ha_codeowners:
+  - '@garbled1'
+ha_config_flow: true
+---
+
+This integration adds "local push" support for [Balboa](http://www.balboawatergroup.com/) Spa WiFi Modules to be used within Home Assistant.
+
+There is currently support for the following device types within Home Assistant:
+
+- Climate
+
+{% include integrations/config_flow.md %}
+
+{% configuration_basic %}
+host:
+  description: Hostname or IP address of your Balboa Spa Wifi Device, e.g., 192.168.1.58
+  required: true
+  type: string
+sync_time:
+  description: Sync the Spa's internal clock with Home Assistant daily
```

Comment:
This is not part of the config flow.

Reply:
Oh, sorry, thats part of the options flow.  Should that be documented in a different section?  I want to let the users know it's possible.

Reply:
That is possible, yes, the `cast` documentation has an example on this:

<https://www.home-assistant.io/integrations/cast#options>